cmake/SetupPackages.cmake

```diff
@@ -82,6 +82,14 @@ endif()
 
 
 if (RAJA_ENABLE_TESTS)
+
+#
+# This conditional prevents build problems resulting from BLT and
+# RAJA each having their own copy of googletest.
+#
+if (RAJA_BUILD_WITH_BLT)
```

Comment:
`RAJA_BUILD_BLT` should probably be added as an option to the top-level CMakeLists.txt, but marked as an advanced option. Kenny did something like this on @LLNL/blt recently.